import.py

```diff
@@ -743,6 +743,13 @@ def main() -> None:
 
     assert isinstance(compiler_type, str)
 
+    if compiler_type is "base":
```

Comment:
`"base"` is fine imo, it's just if the key is unset that we should warn. Also, never use `is` comparisons to compare literals, it's random whether it will work or not.

Maybe we should print `f"Compiler type: {compiler_type}"` if it's set, btw. So like:
```py
compiler_type = settings.get("compiler_type")
if compiler_type is not None:
    assert isinstance(compiler_type, str)
    print(f"Compiler type: {compiler_type}")
else:
    compiler_type = "base"
    print(...)
```

Reply:
okay done